plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -372,6 +378,28 @@ void GSSettingsDlg::UpdateControls()
 		bool ocl = renderer == GSRendererType::DX9_OpenCL || renderer == GSRendererType::DX1011_OpenCL || renderer == GSRendererType::OGL_OpenCL;
 		bool null = renderer == GSRendererType::Null;
 
+		int selected_CRC = SendMessage(GetDlgItem(m_hWnd, IDC_CRC_LEVEL), CB_GETCURSEL, 0, 0);
+
+		if (prev_renderer == GSRendererType::Undefined)
+			prev_renderer = static_cast<GSRendererType>(i);
+		else if (renderer == GSRendererType::OGL_HW && prev_renderer != GSRendererType::OGL_HW) {
+			if (selected_CRC == 3 && non_default_CRC != 3) {
+				SendMessage(GetDlgItem(m_hWnd, IDC_CRC_LEVEL), CB_SETCURSEL, 2, 0);
+				non_default_CRC = 0;
+			} else if (selected_CRC == 2)
+				non_default_CRC = 2;
+
+			prev_renderer = static_cast<GSRendererType>(i);
+		} else if ((renderer == GSRendererType::DX9_HW || renderer == GSRendererType::DX1011_HW) && prev_renderer != GSRendererType::DX9_HW && prev_renderer != GSRendererType::DX1011_HW) {
+			if (selected_CRC == 2 && non_default_CRC != 2) {
```

Comment:
Most of the ``if()`` checks at this file have braces placed on a new line instead of placing them on the same line as the conditional checks, It would be better to have the placement of braces consistent on the file.

So better change,
```
if(condition){
}
```

**to**

```
if(condition)
{
}
```